app/views/candidates/registrations/resend_confirmation_emails/session_expired.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-heading-l">
+      Session expired
+    </div>
+    <p>
+      Your session has expired.
+    </p>
+    <%= link_to 'Restart your application', '/', class: 'govuk-link' %>
```

Comment:
This could link to `candidates_root_path` instead (which is `/candidates/schools` ?